software/nosql/src/main/java/brooklyn/entity/nosql/riak/RiakClusterImpl.java

```diff
@@ -112,6 +122,38 @@ public URI apply(Collection<URI> input) {
                  .fromMembers()
                  .build();
         addEnricher(first);
+        
+        Map<? extends AttributeSensor<? extends Number>, ? extends AttributeSensor<? extends Number>> enricherSetup = 
+            ImmutableMap.<AttributeSensor<? extends Number>, AttributeSensor<? extends Number>>builder()
+                .put(RiakNode.NODE_PUTS, RiakCluster.NODE_PUTS_1MIN_PER_NODE)
```

Comment:
Would be good if the Riak cluster tests were updated to assert that the `NODE_PUTS_1MIN_PER_NODE` etc are being populated (e.g. that the attribute is eventually non-null).